src/test/scala/org/clulab/habitus/variables/TestEntitiesNorms.scala

```diff
@@ -0,0 +1,307 @@
+      "Farmers used two rice cultivars (IR1529 and Jaya) during the 1997 WS, and exclusively IR13240-108-2-2-3 (released as Sahel 108 in Senegal in 1994) during the 1998 DS. All rice was direct-seeded.",
+      Seq(
+        "Date" -> Seq(("1994", "1994-XX-XX")),
+        "DateRange" -> Seq( ("1997 WS", "1997-XX-XX -- 1997-XX-XX"),("1998 DS", "1998-XX-XX -- 1998-XX-XX") )
+      )
+    ),
+    VariableTest(
+      failingTest,
+      //FIXME; AREA SIZE not extracted
+      "sent17",
+      "Average WS T0 yield was high, i.e. 7.3 ha-1 (ranging from 5.0 to 9.4 t ha-1), considering the region’s potential yield of about 9 t ha-1.",
+      Seq(
+        "Quantity" -> Seq(("9 t ha-1", "9.0 t/ha"), ("from 5.0 to 9.4 t ha-1","5.0 -- 9.4 t/ha")),
+        "AreaSize" -> Seq(("7.3 ha-1", "7.3 ha"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent18",
+      "Average DS T0 yield was relatively low, i.e. 4.4 t ha-1 (ranging from 2.5 to 6.0 t ha-1)",
+      Seq(
+        "Quantity" -> Seq(("4.4 t ha-1", "4.4 t/ha"), ("from 2.5 to 6.0 t ha-1", "2.5 -- 6.0 t/ha"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent19",
+      "The favourable climate conditions and the adoption of varieties with shorter cropping cycles allow for two rice harvests per year, namel in the dry and rainy seasons (Van Oort et al., 2016).",
+      Seq(
+        "Date" -> Seq(("2016", "2016-XX-XX"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent20",
+      "In the SRV, production areas are typically larger in the dry season, which brings fewer problems with pests and birds (Tanaka et al., 2015; USDA-GAIN, 2018).",
+      Seq(
+        "Date" -> Seq(("2015", "2015-XX-XX"), ("2018","2018-XX-XX"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent21",
+      "Average yields in SRV theoretically range between 5.0 and 6.0 t ha-1 in the rainy season and between 6.5 and 7.5 t ha-1 in the dry season (SAED, 2019; USDA-GAIN, 2021)",
+      Seq(
+        "Date" -> Seq(("2021", "2021-XX-XX"), ("2019", "2019-XX-XX")),
+        "Quantity" -> Seq(("between 6.5 and 7.5 t ha-1", "6.5 -- 7.5 t/ha"), ("between 5.0 and 6.0 t ha-1", "5.0 -- 6.0 t/ha"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent22",
+      "seeds of the rice variety Sahel 108 are sown, a short- cycle variety (around 125 days)",
+      Seq(
+        "Quantity" -> Seq(("125 days", "125.0 d")),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent23",
+      "Broadcast seeding is carried out by hand on irrigated plots with a 2-5 cm depth sheet of water",
+      Seq(
+        "Quantity" -> Seq(("2-5 cm", "2.0 -- 5.0 cm")),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent24",
+      "average yields for the two seasons assessed of 4832 kg ha− 1 and 7425 kg ha− 1 for the areas under CONV and INT management, respectively.",
+      Seq(
+        "Quantity" -> Seq(("4832 kg", "4832.0 kg"), ("7425 kg", "7425.0 kg"))
```

Comment:
Please adjust the test to have ha-1 in desired (4832 kg ha-1)
In the sentence itself, please adjust the text to not have a space in ha- 1. 
@kwalcock looks like both tika and science parse have issues with processing ha-1 while converting. I will follow up on that in pdf2txt, but I think that should be handled there instead of adjusting rules in processors to account for this.

Reply:
It would be nice to have the PDF for this.  It doesn't seem like they mess up on all instances of what I suspect is ha<sup>-1</sup>.

Reply:
You are right. Will send relevant files via email shortly.

Reply:
I put the file through the Adobe converter and also saved from the PDF to text.  Neither method got the hyphens correct.  I converted from PDF to PS and it doesn't look like these hyphens are glyphs at all.  They are lines drawn or something similar.  I'm not sure that pdf2txt is the right place to fix it, because it seems fairly specific to habitus.  There may need to be a regular expression that looks for " ha ?-?1" that converts to " ha-1".

Reply:
Thanks for checking! Yes, that makes sense. I'll adjust it in habitus.

Reply:
On the other hand, the provenance gets messed up if the reader, habitus here, changes the text internally so that positions don't match the text file.  Perhaps I should do something so that the file is corrected and not just the ephemeral string at runtime.

Reply:
That would be good too! I wonder if this is a one-off thing or there is actually value in handling this weird case in pdf2txt.